core/constants-dev.yaml

```diff
@@ -2,7 +2,7 @@
 k: 2
 
 # Genesis constants
-genesisN: 12
+genesisN: 50000
```

Comment:
My thinking was that it must be changed to `50000` eventually, so why not now? Since it's the condition that triggers the memory usage blow up, keeping it this big is kind of a regression test.